### DISCUSSION TITLE
Styling of each label shown in input field after multiple select

### DISCUSSION THREAD

**Discussion by ssglopes** [mdbootstrap/TW-Elements#2295] [Share ideas & request features]
Using the select component with the multiple option true you are able to select multiple items from this list by ticking the checkboxes. All the checked items will be shown in the top input field and are comma separated. My question is if there is someway to be able to add styling to each of those labels individual in to top input, eg. that way it would be possible to add a border, padding, margin etc. to each label

**Comment by iprzybysz** [comment]
Hi @ssglopes, unfortunately, it's not possible to apply individual styling to the labels within the top input field in our component.

**Reply by ssglopes** [reply]
Hi @iprzybysz  thanks for the feedback. Maybe it could become a feature request for any future release? Have a great day.

**Reply by iprzybysz** [reply]
Thanks for your creative proposal. I've added it to our ideas list.
